translations/messages.fr.xlf

```diff
@@ -3624,6 +3608,14 @@
         <source>Failed! Your opponent can either align:&lt;ul&gt;&lt;li&gt;the squares horizontally on the third line or vertically on the third column,&lt;/li&gt;&lt;li&gt;small pieces horizontally on the first line or vertically on the second or third column,&lt;/li&gt;&lt;li&gt;empty pieces horizontally on the first line or vertically on the first column,&lt;/li&gt;&lt;li&gt;clear pieces diagonally.&lt;/li&gt;&lt;/ul&gt;</source>
         <target>Raté ! Votre adversaire peut soit aligner:&lt;ul&gt;&lt;li&gt;les carrés horizontalement sur la troisième ligne ou verticalement sur la troisième colonne ;&lt;/li&gt;&lt;li&gt;les petites pièces horizontalement sur la première ligne ou verticalement sur la deuxième ou troisième colonne ;&lt;/li&gt;&lt;li&gt;les pièces vides horizontalement sur la première ligne ou verticalement sur la première colonne ;&lt;/li&gt;&lt;li&gt;les pièces claires diagonalement.&lt;/li&gt;&lt;/ul&gt;</target>
       </trans-unit>
+      <trans-unit id="4859304380690720659" datatype="html">
+        <source>A game of Quarto can be configured to make it more complicated for one or both players.&lt;br/&gt;When you create a game and you want to change the original configuration, you can add a level to one or two players. When your level increases, it becomes easier for you to win, so, if the first player is weaker at this game, make the first player play at level two and the game will be more even and interesting.&lt;br/&gt;At level two, a player can win by grouping pieces with a common criterion on a 2x2 square.&lt;br/&gt;In this board, Light can win by making a square.&lt;br/&gt;&lt;br/&gt;You&apos;re playing Light, win.</source>
+        <target>Une partie de Quarto peut être configurée pour la rendre plus compliqué pour un ou deux des joueurs.&lt;br/&gt;Quand vous créez une partie et que vous voulez changer la configuration originale, you pouvez ajouter un niveau à un ou plusieurs joueurs. Quand ton niveau augmente, ça devient plus facile pour vous de gagner, docn, si le premier joueur est plus faible à ce jeu, faite le jouer au niveau deux et cette partie sera plus intéréssante.&lt;br/&gt;Au niveau deux, un joueur peu gagner en groupant des pièces avec un critère commun sur un carré de 2x2.&lt;br/&gt;Sur ce plateau, Clair peu gagner en faisant un carré.&lt;br/&gt;&lt;br/&gt;Vous jouez Clair, gagnez.</target>
```

Comment:
compliqué -> compliquée
you -> vous
ton -> votre
ça -> cela
docn -> donc (puis séparer la phrase avant le "donc")
faite -> faites
cette partie -> la partie

Je te conseille de faire plus attention à la rédaction, de te relire ou de demander à Monsieur GPT de traduire ou de corriger !

Reply:
you pouvez xD
 ton -> votre ?? COMMENT TU SAIS QU'IELSEUX S'IDENTIFIESENT COMME UN THEY THEM ?
 Fait. Ouais cat-i-farted peux aider